src/_posts/platform/app/2000-01-01-monorepo.md

```diff
@@ -0,0 +1,80 @@
+index: 21
+---
+
+Monorepos are a common architectural pattern where multiple projects or components are stored within a single Git repository. Scalingo supports monorepos by allowing you to specify which subdirectory should be deployed for a given application.
+
+## The PROJECT_DIR Environment Variable
+
+By default, Scalingo's deployment system looks for the application's entry point (e.g., `package.json`, `Gemfile`, `requirements.txt`) at the root of the Git repository.
+
+In a monorepo setup, you need to tell Scalingo which directory contains the application you want to deploy. You can do this by setting the `PROJECT_DIR` environment variable.
+
+### How it Works
+
+When `PROJECT_DIR` is set:
+1. The build system changes its working directory to the specified folder.
+2. Technology detection (buildpack selection) is performed within that directory.
+3. Only the content of this directory is included in the final application image.
+
+{% warning %}
+Only the `PROJECT_DIR` directory will be present during runtime. Parent directories or sibling directories are **not** included in the application image.
+{% endwarning %}
+
+## Example Setup
+
+Imagine a repository with the following structure:
+
+```text
+.
+├── backend/
+│   ├── package.json
+│   └── src/
+├── frontend/
+│   ├── package.json
+│   └── src/
+└── README.md
+```
+
+To deploy the backend and frontend as separate Scalingo applications from this single repository:
+
+1. **Create two applications** on Scalingo (e.g., `my-app-api` and `my-app-ui`).
+2. **Connect both applications** to the same Git repository.
+3. **Configure the environment** for each application:
+    - For `my-app-api`: Set `PROJECT_DIR=backend`
+    - For `my-app-ui`: Set `PROJECT_DIR=frontend`
+
+The next time you push to the repository, each application will build and deploy only its respective subdirectory.
+
+## Using a Manifest File
+
+If you prefer to keep your configuration within your repository, you can also specify the `PROJECT_DIR` in a [`scalingo.json` manifest file]({% post_url platform/app/2000-01-01-app-manifest %}) located at the root of your repository.
+
+{% note %}
+The `scalingo.json` manifest is only used for [review apps]({% post_url
+platform/app/2000-01-01-review-apps %}) and [one-click deployments]({%
+post_url platform/deployment/2000-01-01-one-click-deploy %}).
+
+For existing applications, adding or modifying `PROJECT_DIR` in the manifest will have no effect. You must set it as an
+environment variable using the dashboard or the CLI.
+{% endnote %}
+
+```json
+{
+  "name": "my-app-api",
+  "env": {
+    "PROJECT_DIR": {
+      "value": "backend"
+    }
+  }
+}
+```
+
+## Common Use Cases
+
+- **Backend/Frontend Separation**: Keeping your API and client-side code in the same repo but deploying them to different environments.
+- **Microservices**: Managing multiple microservices in a single repository.
+- **Documentation**: Keeping your documentation site alongside your application code.
```

Comment:
suggestion: I'm not sure this is useful. Either a customer has a monorepo and they already know why they do that. Or they don't have a monorepo and they won't look into Scalingo documentation about that